src/test/storm/modelchecker/prctl/mdp/DiscountingMdpPrctlModelCheckerTest.cpp

```diff
@@ -0,0 +1,389 @@
+#include "test/storm_gtest.h"
```

Comment:
Can this be merged with the MdpPrctlModelCheckerTest to reduce code duplications? (Same for the DTMC variant)?
I know that some environments there don't apply but these could be skipped if they don't work

Reply:
I think we should find a way to get rid of those massive code duplications in the tests. The `Lra*Test.cpp` classes are guilty of this, too. But that's something for a different PR :)